rust_dev_preview/examples/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 # This should be kept in sync with https://github.com/awslabs/aws-sdk-rust#supported-rust-versions-msrv
 [toolchain]
-channel = "1.69.0"
+channel = "stable"
```

Comment:
Either remove/update the comment or set this to `1.70.0`

Reply:
Ack - resolved in base PR https://github.com/awsdocs/aws-doc-sdk-examples/pull/5434